app/models/planning_application.rb

```diff
@@ -303,10 +301,6 @@ def agent_or_applicant_name
     end
   end
 
-  def policy_classes_editable
```

Comment:
`policy_classes_changed?` obviously breaks now it's an association rather than a column, and there's a check in `PolicyClassesController` that makes this redundant so I think it's safe to remove.